custom-connectors/HR Cloud/apiDefinition.swagger.json

```diff
@@ -0,0 +1,961 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "HR Cloud",
+    "description": "HR Cloud is a powerful and globally recognized HRMS software for the entire employee lifecycle.\nUser this connector to easily send employee information between HR Cloud and other systems you use.",
+    "version": "1.0",
+	"contact":{
+	"name": "HR Cloud Support",
+	"url": "https://support.hrcloud.com/en/help-center",
+	"email": "support@hrcloud.com"
+	}
+  },
+  "host": "corehr-api.hrcloud.com",
+  "basePath": "/",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [],
+  "produces": [],
+  "paths": {
+    "/v1/cloud/xEmployee": {
+      "get": {
+        "responses": {
+          "200": {
+            "description": "default",
+            "schema": {
+              "type": "array",
+              "items": {
+                "type": "object",
+                "properties": {
+                  "Id": {
+                    "type": "string",
+                    "description": "Id"
+                  },
+                  "New_custom_field_on_employee": {
+                    "type": "string",
+                    "description": "New_custom_field_on_employee"
+                  },
+                  "xAddress1": {
+                    "type": "string",
+                    "description": "xAddress1"
+                  },
+                  "xAddress2": {
+                    "type": "string",
+                    "description": "xAddress2"
+                  },
+                  "xBonus": {
+                    "type": "object",
+                    "properties": {
+                      "Amount": {
+                        "type": "string",
+                        "description": "Amount"
+                      },
+                      "Currency": {
+                        "type": "string",
+                        "description": "Currency"
+                      }
+                    },
+                    "description": "xBonus"
+                  },
+                  "xCellPhone": {
+                    "type": "string",
+                    "description": "xCellPhone"
+                  },
+                  "xCity": {
+                    "type": "string",
+                    "description": "xCity"
+                  },
+                  "xCountry": {
+                    "type": "string",
+                    "description": "xCountry"
+                  },
+                  "xCreatedOn": {
+                    "type": "string",
+                    "description": "xCreatedOn"
```

Comment:
@HRCIntegrationsTeam All these description values of the properties looks like IDs, Can you please provide valid description of them.